flask_dynamo/manager.py

```diff
@@ -42,8 +42,10 @@ def wait_not_exists(self, table_name):
         self._wait(table_name, 'table_not_exists')
 
     def create_all(self, wait=False):
+        tables_name_list = [table.name for table in self._connection.tables.all()]
         for table in self._table_config:
-            self._connection.create_table(**table)
+            if table['TableName'] not in tables_name_list:
+                self._connection.create_table(**table)
         if wait:
```

Comment:
This wait could be moved under the if at https://github.com/rdegges/flask-dynamo/pull/26/files#diff-2c5bcf58c2ebfb56c8d985f32ea7a069R47 ; because if it already exists, no reason to wait...